src/main/kotlin/no/nav/syfo/client/moteplanlegger/domain/PlanlagtMoteDTO.kt

```diff
@@ -48,15 +48,18 @@ fun PlanlagtMoteDTO.toNewDialogmotePlanlagt(
         opprettetAv = requestByNAVIdent,
         arbeidstaker = NewDialogmotedeltakerArbeidstaker(
             personIdent = PersonIdentNumber(this.fnr),
+            fritekstInnkalling = ""
```

Comment:
Hvorfor sette disse verdiene til tom string? Skal feltene være påkrevde / not nullable? I så fall kan man kanskje sette default verdi "" i selve DTOene, evnt ved lagring til base?

Reply:
Ja, skal se om dette kan forbedres. Har forsåvidt håndtert det i database-skjemaet (NOT NULL DEFAULT ''), men litt usikker på om jeg har lyst til å ha dem definert med ? i dto'en.

Reply:
Skjønner :) Ikke noe stort problem altså, bare  tenkte over det

Reply:
Nå er jeg litt usikker på om PlanlagtMoteDTO skal brukes framover (det var denne og tilhørende post-tjeneste vi snakket om at kanskje kan fjernes). Det er den andre DTO'en dere bør ta sikte på å bruke fra klientene nå (dvs NewDialogmoteDTO).